test/test.py

```diff
@@ -7,160 +7,240 @@
 import os
 
 
+class AudioBackendScope:
```

Comment:
This would be convenient in general, but I wasn't planning to introduce this as a feature yet since torchvision doesn't have this.